otx/algorithms/segmentation/configs/ocr_lite_hrnet_18/compression_config.json

```diff
@@ -22,10 +22,14 @@
       "lr": 1e-4
     },
     "lr_config": {
+      "_delete_": true,
+      "policy": "customstep",
       "fixed": null,
       "fixed_iters": 0,
+      "fixed_ratio": 10.0,
       "warmup": null,
       "warmup_iters": 0,
+      "warmup_ratio": 0.3333,
```

Comment:
Where were the updates from?

Reply:
Current OTX segmentation recipe does not use `CustomstepLrUpdateHook` but `ReduceLROnPlateauLrUpdaterHook`. The configuration presented here is `CustomstepLrUpdateHook` though. `fixed_ratio` and `warmup_ratio` parts were removed in https://github.com/openvinotoolkit/training_extensions/pull/1400/commits/12e6951e58924c1885357b0729ce975a9f6e0229 as they were just for test purpose.